tests/memgraph/test_query_builder.py

```diff
@@ -283,6 +287,341 @@ def test_load_csv_no_header(memgraph):
     mock.assert_called_with(expected_query)
 
 
+def test_where_literal(memgraph):
+    query_builder = (
+        QueryBuilder()
+        .match()
+        .node("L1", variable="n")
+        .to("TO")
+        .node("L2", variable="m")
```

Comment:
keyword argument missing.

Also in all tests after this one, the same keyword arguments are missing